game/src/camera.rs

```diff
@@ -46,21 +48,28 @@ pub struct MainCamera;
 pub struct GameViewLimits(Rect);
 
 fn setup_main_camera(mut commands: Commands) {
-    let mut camera = Camera2dBundle::default();
-
+    let mut camera = Camera2dBundle {
+        camera: Camera {
+            hdr: true,
+            ..default()
+        },
+        tonemapping: Tonemapping::AcesFitted,
```

Comment:
Just curious, why did you choose the AcesFitted tonemapping function?

Reply:
I forget why I chose this originally, I think because of the brighter colors or something seemed to look better then default; might change in the future to see what looks best